sonic_platform_base/module_base.py

```diff
@@ -17,6 +17,18 @@ class ModuleBase(device_base.DeviceBase):
     # Device type definition. Note, this is a constant.
     DEVICE_TYPE = "module"
 
+    # Possible card types
+    MODULE_TYPE_CONTROL = "CONTROL-CARD"
+    MODULE_TYPE_LINE    = "LINE-CARD"
+    MODULE_TYPE_FABRIC  = "FABRIC-CARD"
+
+    # Possible card status
+    MODULE_STATUS_EMPTY   = "Empty"
+    MODULE_STATUS_OFFLINE = "Offline"
+    MODULE_STATUS_PRESENT = "Present"
+    MODULE_STATUS_FAULT   = "Fault"
+    MODULE_STATUS_ONLINE  = "Online"
```

Comment:
can you clarify the meaning of each status in the description.

Reply:
Added.